README.md

```diff
@@ -1 +1,84 @@
-What?

The idea of the software is to make the user control the vehicle's direction and speed using a mobile application. In addition, the user can call the vehicle to self-drive to them. If the vehicle gets around any obstacles on its way, it avoids hitting them and moves around them.

Why?

The goal is to make it more efficient for the users with their daily tasks by calling their cars to them in parking lot, instead of looking for them.

How?

The users can connect the vehicle to their devices via Bluetooth, and they can do that through the mobile app. The joysticks/buttons screen on the app will help the user control the direction and speed of the vehicle. The user can get the vehicle to self-drive to them using GPS coordinates streamed from their phones. The sensors in the vehicle will help avoiding hitting any possible objects on its way.

Software:

- Java
- JavaFX
- Github
- Arduino IDE

Hardware :

- Smartphone
- 1x Smartcar
- 8x AA rechargeable batteries
- 1x battery charger
- 2x micro-LIDAR sensors
- 2x ultrasonic sensors
- 1x MicroUSB cable
- 360� servo motors
- PIR motion detectors
- Infrared sensors (line following)
- Infrared sensors (distance)
- ADXL337 accelerometers
- ZX Gesture sensors
\ No newline at end of file
+[1. Project Overview](https://github.com/DIT112-V20/group-01#project-overview)
+* [what it the idea of our project?](https://github.com/DIT112-V20/group-01#what-is-the-idea-of-our-project)
+* [Why will we make it?](https://github.com/DIT112-V20/group-01#why-will-we-make-it)
```

Comment:
You don't have to include the entire URL Just the part with the `#why-will-we-make-it`should be enough.